### PR TITLE
feat(components): expansion panel web component

### PR DESCRIPTION
**Pull request by carolineguzylak**
## Description

Developed expansion panel component using Lit framework - implemented in storybook.

### What's included?

- Expansion panel component
- Expansion panel documentation page

#### Test Steps

- [ ] `npm run storybook`

#### General Tests for Every PR

- [ ] `npm run start` still works.
- [ ] `npm run lint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build` still works.

##### Screenshots or link to StackBlitz/Plunker

Screenshot 2023-08-30 at 11 17 04 AM
Screenshot 2023-08-30 at 11 17 21 AM

### CODE REVIEW

File: libs/components/src/expansion-panel/cv-expansion-panel.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    let toggledPanelIndex = e.detail.index;
+    let panel = this.panelItems[toggledPanelIndex];
+    let open = panel.open;
+
+    // close the currently open panel if there is one
+    this.panelItems.forEach((item) => {
+      item.resetPanel();
+    });
+
+    if (open) {
```
Comment (owilliams320) [inline libs/components/src/expansion-panel/cv-expansion-panel.ts:42]: does this do anything? looks like panel.open would already be true from above if the open variable is already assigned from panel.open?

**Review by owilliams320 — CHANGES_REQUESTED** [review]

**Comment by owilliams320** [comment]
@carolineguzylak looks like there is a few lint and build issues. You should be able to run locally using `nx affected build` and `nx affected lint`

File: libs/components/src/expansion-panel/cv-expansion-panel.ts (hunk 1 above)
Comment (carolineguzylak) [reply libs/components/src/expansion-panel/cv-expansion-panel.ts:35]: I removed this

**Review by owilliams320 — CHANGES_REQUESTED** [review]
One comment then looks good

**Comment by owilliams320** [comment]
@carolineguzylak looks like the pipeline did not pass you should be able run `npm run lint -- --fix`

File: libs/components/src/expansion-panel/cv-expansion-panel-item.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import { LitElement, css, html, unsafeCSS } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import styles from './cv-expansion-panel-item.scss?inline';
+import '../icon/icon';
+import '../action-ribbon/action-ribbon';
```
[the remark this answers is not in the document]
Comment (carolineguzylak) [reply libs/components/src/expansion-panel/cv-expansion-panel-item.ts:5]: Yes

**Comment by carolineguzylak** [comment]
> @carolineguzylak looks like the pipeline did not pass you should be able run `npm run lint -- --fix`

**Review by owilliams320 — APPROVED** [review]
LGTM! 🏎️

**Pull request merged**